external/dhcpc/dhcpc.c

```diff
@@ -659,10 +674,14 @@ static int dhcpc_request(void *handle, struct dhcpc_state *presult)
 		 (presult->netmask.s_addr >> 8) & 0xff,
 		 (presult->netmask.s_addr >> 16) & 0xff,
 		 (presult->netmask.s_addr >> 24) & 0xff);
-	ndbg("Got DNS server %d.%d.%d.%d\n", (presult->dnsaddr.s_addr) & 0xff,
-		 (presult->dnsaddr.s_addr >> 8) & 0xff,
-		 (presult->dnsaddr.s_addr >> 16) & 0xff,
-		 (presult->dnsaddr.s_addr >> 24) & 0xff);
+
+	for (int i = 0; i < total_dns; ++i) {
```

Comment:
Iterator "i" for the 'for' loop should be defined at the start of the function with other variables.